modules/vulnerability-management20-creating-report.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-creating-report_{context}"]
+= Creating vulnerability management report configurations
+
+[role="_abstract"]
+{product-title-short} guides you through the process of creating a vulnerability management report configuration. This configuration determines the information that will be included in a report job that runs at a scheduled time or that you run on demand.
+
+.Procedure
+. On the {product-title-short} portal, navigate to *Vulnerability Management 2.0* -> *Vulnerability Reporting*.
+. Click *Create report*.
+. Enter a name for your report configuration in the *Report name* field.
+. Optional: Enter text describing the report configuration in the *Description* field.
+. In the *CVE severity* field, select the severity of common vulnerabilities and exposures (CVEs) that you want to include in the report configuration. ##Do we need to clarify where the severity indicators come from? Red Hat Product Security severity? Or other? I see this question sometimes comes up in slack/gchat so wondered if we need to mention it.##
```

Comment:
We can compile all the common vulnerability management keywords in a glossary as a part of GA release

Reply:
Added a [Jira ticket](https://issues.redhat.com/browse/ROX-19538) to track separately.